cmake/IgnUtils.cmake

```diff
@@ -150,6 +155,79 @@ macro(ign_find_package PACKAGE_NAME)
   # Parse the arguments
   _ign_cmake_parse_arguments(ign_find_package "${options}" "${oneValueArgs}" "${multiValueArgs}" ${ARGN})
 
+  #------------------------------------
+  # Handle ticktock
+  # TODO(CH3): Remove on tock
+  if(${PACKAGE_NAME} MATCHES "^ign(ition)?-")
+    message(DEPRECATION
+      "\nThe use of 'ign-' and 'ignition-' prefixed libraries has been "
```

Comment:
has `find_package(ign-*)` ever worked? I thought we always had to spell it out as 1ignition-*`. If that is the case, I would simplify this to only cover the case of `ignition-` to avoid confusion